universum/modules/vcs/swarm.py

```diff
@@ -193,14 +194,13 @@ def report_start(self, report_text):
         self.post_comment(report_text)
 
     def code_report_to_review(self, report):
-        for path, issues in report.items():
-            abs_path = os.path.join(self.client_root, path)
-            if abs_path in self.mappings_dict:
-                for issue in issues:
-                    self.post_comment(issue['message'],
-                                      filename=self.mappings_dict[abs_path],
-                                      line=issue['line'],
-                                      no_notification=True)
+        with Uninterruptible(self.out.log_error) as run:
```

Comment:
What is the benefit of using context manager in comparison to handling one exception?